### DISCUSSION TITLE
Brotli Compression

### DISCUSSION THREAD

**Discussion by jadamconnor** [tailwindlabs/tailwindcss#7058] [Help]
I'm using Tailwind in a Next.js project, and I'm optimizing for compression. The official documentation suggests minifying with nanocss and compression with brotli. For the former, we are given and example. But the brotli suggestion just leads to the brotli Wikipedia page. Is anyone aware of any examples of brotli being used for compression of Tailwind?

**Comment by adamwathan** [comment]
Hey! Brotli compression happens in your web server, and is usually configured for you automatically by the platform you are deploying your site to. For example I have never in my life configured Brotli and every site I run uses it.

Vercel configures it automatically for example, as does Netlify, Cloudflare, etc. If you do need to configure it manually the steps will be specific to your web server, so if you are maintaining your own nginx server for example you’ll want to search for configuring Brotli with nginx.

**Reply by jadamconnor** [reply]
Hey, thanks for the response! It turns out you're right (of course). I'm just a little confused because I see in the response headers that the content-encoding is in fact 'br,' but the sources in dev tools shows a size of ~83kb (79kb of which is unused) for my css bundle (/_next/static/css/6a3d4c1d5e7603e6.css). That seems large for compression. That is after minification with cssnano by the way.

**Reply by jackson-jackson** [reply]
Did you ever figure this out? I'm having the same issue in my SvelteKit Vite app.
